<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  
  <title>Biolaosun</title>
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
  <meta property="og:type" content="website">
<meta property="og:title" content="Biolaosun">
<meta property="og:url" content="https://sun010848.github.io/index.html">
<meta property="og:site_name" content="Biolaosun">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Biolaosun">
  
    <link rel="alternate" href="/atom.xml" title="Biolaosun" type="application/atom+xml">
  
  
    <link rel="icon" href="/favicon.png">
  
  
    <link href="//fonts.googleapis.com/css?family=Source+Code+Pro" rel="stylesheet" type="text/css">
  
  <link rel="stylesheet" href="/css/style.css">
  

</head>

<body>
  <div id="container">
    <div id="wrap">
      <header id="header">
  <div id="banner"></div>
  <div id="header-outer" class="outer">
    <div id="header-title" class="inner">
      <h1 id="logo-wrap">
        <a href="/" id="logo">Biolaosun</a>
      </h1>
      
    </div>
    <div id="header-inner" class="inner">
      <nav id="main-nav">
        <a id="main-nav-toggle" class="nav-icon"></a>
        
          <a class="main-nav-link" href="/">Home</a>
        
          <a class="main-nav-link" href="/archives">Archives</a>
        
      </nav>
      <nav id="sub-nav">
        
          <a id="nav-rss-link" class="nav-icon" href="/atom.xml" title="RSS Feed"></a>
        
        <a id="nav-search-btn" class="nav-icon" title="Search"></a>
      </nav>
      <div id="search-form-wrap">
        <form action="//google.com/search" method="get" accept-charset="UTF-8" class="search-form"><input type="search" name="q" class="search-form-input" placeholder="Search"><button type="submit" class="search-form-submit">&#xF002;</button><input type="hidden" name="sitesearch" value="https://sun010848.github.io"></form>
      </div>
    </div>
  </div>
</header>
      <div class="outer">
        <section id="main">
  
    <article id="postc-20171017_VEP_annotation2" class="article article-type-postc" itemscope itemprop="blogPost">
  <div class="article-meta">
    <a href="/2017/10/16/20171017_VEP_annotation2/" class="article-date">
  <time datetime="2017-10-15T16:00:00.000Z" itemprop="datePublished">2017-10-16</time>
</a>
    
  </div>
  <div class="article-inner">
    
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2017/10/16/20171017_VEP_annotation2/">VEP--强大的变异注释工具</a>
    </h1>
  

      </header>
    
    <div class="article-entry" itemprop="articleBody">
      
        <p class="author-title">(本文由GeneDock公司 Bioinformatics Engineer 刘轩竹撰写，转载请保留作者信息和原文链接)</p>

<h2 id="背景"><a href="#背景" class="headerlink" title="背景"></a>背景</h2><p><span style="font-size:16px">随着基因测序技术的快速发展，检测成本大幅度降低，使得基因组测序投入临床市场并服务大众。但二代测序发现的变异数量是巨大的，造成变异解读及查找致病基因需耗费大量的人力物力。以人类基因组为例，1个人类基因组包含近3500000个SNV突变和1000个拷贝数变异，其中约20000-25000个变异是在编码区，10000个位点改变了氨基酸编码，但仅有50-100个突变造成蛋白截断或功能缺失，手动检查每个位点非常耗时且有些不切实际。</span></p>
<p><span style="font-size:16px">Variant Effect Predictor（VEP）是功能强大的注释、分析工具。它可以对二代测试产生的不同类型变异进行注释，包含SNPs, insertions, deletions, copy number variants和structural variants。也可以依据各种数据库的内容，根据需要，对变异进行过滤和排序。</span> </p>
<p><span style="font-size:16px">GeneDock公有云平台目前提供VEP注释的工作流，使用VEP88对变异进行注释，输入变异检测后的vcf文件，输出注释后的文件（vcf格式）和经筛选后的变异位点。</span></p>
        
          <p class="article-more-link">
            <a href="/2017/10/16/20171017_VEP_annotation2/#more">Read More</a>
          </p>
        
      
    </div>
    <footer class="article-footer">
      <a data-url="https://sun010848.github.io/2017/10/16/20171017_VEP_annotation2/" data-id="cj8wkc2ex0000tls6xxw9ic5s" class="article-share-link">Share</a>
      
      
    </footer>
  </div>
  
</article>


  
    <article id="postc-20170916_Sequencing_QC_Stat" class="article article-type-postc" itemscope itemprop="blogPost">
  <div class="article-meta">
    <a href="/2017/09/15/20170916_Sequencing_QC_Stat/" class="article-date">
  <time datetime="2017-09-14T16:00:00.000Z" itemprop="datePublished">2017-09-15</time>
</a>
    
  </div>
  <div class="article-inner">
    
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2017/09/15/20170916_Sequencing_QC_Stat/">测序数据质控统计--微不足道？意义重大！</a>
    </h1>
  

      </header>
    
    <div class="article-entry" itemprop="articleBody">
      
        <p class="author-title">(本文由GeneDock公司 Bioinformatics Engineer 武雅蓉撰写，转载请保留作者信息和原文链接)</p>

<h2 id="引言："><a href="#引言：" class="headerlink" title="引言："></a>引言：</h2><p>随着NGS测序成本的降低，高通量测序分析变得越来越普遍。然而，在实际工作中，生信人员往往拿到测序公司提供的数据之后，就直接开始跑流程，去接头、过滤、鉴定变异位点…</p>
<p>然而，过滤前和过滤后的数据到底有什么不同？数据中N碱基的含量如何？低质量的数据是否居多？测序深度是否达到要求？靶标区域覆盖度如何？这一系列的问题往往被急于出分析结果的生信人员（尤其在大Boss的问[逼]候[迫]下）抛之脑后。</p>
<p>满心欢喜去交付自己麻溜完成的分析成果，却遭到了客户对结果的质疑。苦苦回溯分析过程，查找众多可能的出错步骤，最终发现是测序数据质量不过关。前功尽弃，何其郁闷哉…</p>
<p>为保证分析结果的可靠性，对测序数据进行质量控制统计分析意义重大。尤其在临床二代测序检测领域，分析结果的可靠性与否关乎医生的诊断和病人的安危。</p>
<p>参照《临床分子病理实验室二代基因测序检测专家共识》、《二代测序（NGS）技术应用于临床肿瘤精准医学诊断的共识》中要求的指标，以及实际测序数据分析经验，GeneDock的小伙伴为全外显子组和全基因组产品增加了质控统计步骤，主要包括：（1）过滤前后FASTQ文件的基本测序质量统计；（2）比对后BAM文件的基本统计。</p>
<h2 id="FASTQ文件的基本测序质量统计"><a href="#FASTQ文件的基本测序质量统计" class="headerlink" title="FASTQ文件的基本测序质量统计"></a>FASTQ文件的基本测序质量统计</h2><p>###1. <strong>基本统计表格</strong></p>
<table>
<thead>
<tr>
<th style="text-align:left">统计指标</th>
<th style="text-align:left">描述</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">Number of Reads</td>
<td style="text-align:left">reads数目</td>
</tr>
<tr>
<td style="text-align:left">Data Size</td>
<td style="text-align:left">碱基数量</td>
</tr>
<tr>
<td style="text-align:left">N of fq1</td>
<td style="text-align:left">reads1中N碱基数目</td>
</tr>
<tr>
<td style="text-align:left">N of fq2</td>
<td style="text-align:left">reads2中N碱基数目</td>
</tr>
<tr>
<td style="text-align:left">Low qual base of fq1(&lt;=15)</td>
<td style="text-align:left">reads1中低质量的碱基数目</td>
</tr>
<tr>
<td style="text-align:left">Low qual base of fq2(&lt;=15)</td>
<td style="text-align:left">reads2中低质量的碱基数目</td>
</tr>
<tr>
<td style="text-align:left">Q20 of fq1</td>
<td style="text-align:left">reads1中质量值&gt;=20的碱基所占的比例</td>
</tr>
<tr>
<td style="text-align:left">Q20 of fq2</td>
<td style="text-align:left">reads2中质量值&gt;=20的碱基所占的比例</td>
</tr>
<tr>
<td style="text-align:left">Q30 of fq1</td>
<td style="text-align:left">reads1中质量值&gt;=30的碱基所占的比例</td>
</tr>
<tr>
<td style="text-align:left">Q30 of fq2</td>
<td style="text-align:left">reads2中质量值&gt;=30的碱基所占的比例</td>
</tr>
<tr>
<td style="text-align:left">GC of fq1</td>
<td style="text-align:left">reads1的GC含量</td>
</tr>
<tr>
<td style="text-align:left">GC of fq2</td>
<td style="text-align:left">reads2的GC含量</td>
</tr>
<tr>
<td style="text-align:left">Error of fq1</td>
<td style="text-align:left">reads1的错误率</td>
</tr>
<tr>
<td style="text-align:left">Error of fq2</td>
<td style="text-align:left">reads2的错误率</td>
</tr>
<tr>
<td style="text-align:left">Discard Reads related to N and low qual</td>
<td style="text-align:left">N碱基和低质量的reads所占比例</td>
</tr>
<tr>
<td style="text-align:left">Discard Reads related to Adapter</td>
<td style="text-align:left">带接头的reads比例</td>
</tr>
</tbody>
</table>
<p>###2. <strong>质量值和GC含量图</strong><br>1) <strong>碱基含量分布图</strong></p>
<p>根据reads中每个位置碱基 A、T、G、C、N 所占的比例，绘制碱基含量分布图。根据碱基互补原理，A和T、G和C的含量应该基本一致，前几bp出现抖动情况是由于随机引物、测序反应开始酶和底物结合不太稳定导致，属于测序本身所带来的正常抖动。碱基含量分布与建库、测序和物种相关。</p>
<center><img src="https://cdn.genedock.com/img/blog/clean_merge_GC.png" alt=""></center>


<center><strong>图1：碱基含量分布图</strong></center>


<center>图中，横坐标为reads中的碱基位置，纵坐标为碱基所占的比例，不同颜色代表不同的碱基类型，左右两侧分别为双端测序序列两条reads的碱基分布情况。</center>

<p>2) <strong>测序质量分布图</strong></p>
<p>根据reads中每个位置碱基的平均测序质量值，绘制测序质量分布图。测序片段末端的碱基质量一般会比前端的低，但测序质量主要会分布在 Q20 以上，才能为后续数据分析提供保证。</p>
<center><img src="https://cdn.genedock.com/img/blog/clean_merge_QM.png" alt=""></center>


<center><strong>图2：测序质量分布图</strong></center>


<center>图中，横坐标为reads中的碱基位置，纵坐标为每个位置的平均测序质量值，左右两侧分别为双端测序序列两条reads的质量值分布情况。</center>

<p>3) <strong>碱基总体质量分布图</strong></p>
<p>根据不同测序质量值的碱基数目情况，绘制碱基总体质量分布图。对于二代测序，一般要求达到Q20的碱基比例 &gt;95%（最差 &gt;=90%），Q30的碱基比例 &gt;85%（最差 &gt;=80%）。</p>
<center><img src="https://cdn.genedock.com/img/blog/clean_merge_QD.png" alt=""></center>


<center><strong>图3：碱基总体质量分布图</strong></center>


<center>图中，横坐标为测序质量值，纵坐标为该质量值处的碱基数目。</center>

<p>4) <strong>测序错误率分布图</strong></p>
<p>测序错误率与碱基质量负相关，受测序仪本身、测序试剂、样品等多个因素共同影响。一般来说，测序片段末端的错误率会偏高。</p>
<center><img src="https://cdn.genedock.com/img/blog/clean_merge_Error.png" alt=""></center>


<center><strong>图4：测序错误率分布图</strong></center>


<center>图中，横坐标为reads中的碱基位置，纵坐标为单碱基错误率，左右两侧分别为双端测序序列两条reads的测序错误率分布情况。</center>

<h2 id="比对后BAM文件的基本统计"><a href="#比对后BAM文件的基本统计" class="headerlink" title="比对后BAM文件的基本统计"></a>比对后BAM文件的基本统计</h2><p>###1. <strong>基本统计表格</strong></p>
<table>
<thead>
<tr>
<th style="text-align:left">统计指标</th>
<th style="text-align:left">描述</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">Total_reads_num_in_bam</td>
<td style="text-align:left">bam文件中reads数目</td>
</tr>
<tr>
<td style="text-align:left">Total_reads_num_in_bam_on_target</td>
<td style="text-align:left">bam文件中属于目标区域的reads数目</td>
</tr>
<tr>
<td style="text-align:left">Mapped_reads_num</td>
<td style="text-align:left">比对上的reads数目</td>
</tr>
<tr>
<td style="text-align:left">Mapped_reads_num_on_target</td>
<td style="text-align:left">目标区域比对上的reads数目</td>
</tr>
<tr>
<td style="text-align:left">Mapping_rate(%)</td>
<td style="text-align:left">比对率</td>
</tr>
<tr>
<td style="text-align:left">Mapping_rate_on_target(%)</td>
<td style="text-align:left">目标区域比对率</td>
</tr>
<tr>
<td style="text-align:left">Duplication_reads_num</td>
<td style="text-align:left">重复的reads数目</td>
</tr>
<tr>
<td style="text-align:left">Duplication_reads_num_on_target</td>
<td style="text-align:left">目标区域重复的reads数目</td>
</tr>
<tr>
<td style="text-align:left">Duplication_rate(%)</td>
<td style="text-align:left">reads重复率</td>
</tr>
<tr>
<td style="text-align:left">Duplication_rate_on_target(%)</td>
<td style="text-align:left">目标区域reads重复率</td>
</tr>
<tr>
<td style="text-align:left">Mapped_bases_num</td>
<td style="text-align:left">比对上的碱基数目</td>
</tr>
<tr>
<td style="text-align:left">Mapped_bases_num_on_target</td>
<td style="text-align:left">目标区域比对上的碱基数目</td>
</tr>
<tr>
<td style="text-align:left">Mapped_bases_num (cigar)</td>
<td style="text-align:left">CIGAR中比对上的碱基数目</td>
</tr>
<tr>
<td style="text-align:left">Mapped_bases_num_on_target (cigar)</td>
<td style="text-align:left">目标区域CIGAR中比对上的碱基数目</td>
</tr>
<tr>
<td style="text-align:left">Mismatch_num</td>
<td style="text-align:left">错配的碱基数目</td>
</tr>
<tr>
<td style="text-align:left">Mismatch_num_on_target</td>
<td style="text-align:left">目标区域错配的碱基数目</td>
</tr>
<tr>
<td style="text-align:left">Mismatch_rate(%)</td>
<td style="text-align:left">碱基错配率</td>
</tr>
<tr>
<td style="text-align:left">Mismatch_rate_on_target(%)</td>
<td style="text-align:left">目标区域碱基错配率</td>
</tr>
<tr>
<td style="text-align:left">Average_read_length(bp)</td>
<td style="text-align:left">平均reads长度</td>
</tr>
<tr>
<td style="text-align:left">Target_region_size(bp)</td>
<td style="text-align:left">目标区域长度</td>
</tr>
<tr>
<td style="text-align:left">BaseNum_covered_on_target(bp)</td>
<td style="text-align:left">目标区域覆盖的碱基数目</td>
</tr>
<tr>
<td style="text-align:left">Coverage_of_target_region(%)</td>
<td style="text-align:left">目标区域覆盖率</td>
</tr>
<tr>
<td style="text-align:left">Average_sequencing_depth_on_target</td>
<td style="text-align:left">目标区域平均测序深度</td>
</tr>
<tr>
<td style="text-align:left">Fraction_of_target_covered_with_at_least_150x(%)</td>
<td style="text-align:left">目标区域测序深度&gt;=150X的百分比</td>
</tr>
<tr>
<td style="text-align:left">Fraction_of_target_covered_with_at_least_100x(%)</td>
<td style="text-align:left">目标区域测序深度&gt;=100X的百分比</td>
</tr>
<tr>
<td style="text-align:left">Fraction_of_target_covered_with_at_least_50x(%)</td>
<td style="text-align:left">目标区域测序深度&gt;=50X的百分比</td>
</tr>
<tr>
<td style="text-align:left">Fraction_of_target_covered_with_at_least_30x(%)</td>
<td style="text-align:left">目标区域测序深度&gt;=30X的百分比</td>
</tr>
<tr>
<td style="text-align:left">Fraction_of_target_covered_with_at_least_20x(%)</td>
<td style="text-align:left">目标区域测序深度&gt;=20X的百分比</td>
</tr>
<tr>
<td style="text-align:left">Fraction_of_target_covered_with_at_least_10x(%)</td>
<td style="text-align:left">目标区域测序深度&gt;=10X的百分比</td>
</tr>
</tbody>
</table>
<p>###2. <strong>测序深度图</strong></p>
<p>1) <strong>目标区域测序深度图</strong></p>
<p>根据目标区域每个位点的覆盖度，绘制测序深度分布图，可直观展示平均测序深度情况，同时用来衡量目标区域测序的随机性。当平均测序深度和峰值重合时，测序的随机性较好。</p>
<center><img src="https://cdn.genedock.com/img/blog/histPlot.png" alt=""></center>


<center><strong>图5：目标区域测序深度图</strong></center>


<center>图中，横坐标为测序深度，纵坐标为该测序深度碱基位点所占的百分比，其中红色和黄色箭头分别代表测序深度平均值和中位数。</center>


<p>2) <strong>测序深度累积曲线</strong></p>
<p>根据测序深度累积曲线，可直观得到大于某测序深度时的碱基覆盖率。和目标区域测序深度图一样，可用于指导后续数据分析时的参数的设置。例如：检测SNP时，至少需要多少reads的支持。</p>
<center><img src="https://cdn.genedock.com/img/blog/cumuPlot.png" alt=""></center>


<center><strong>图6：测序深度累积曲线</strong></center>


<center>图中，横坐标为累积型测序深度，纵坐标为累积型测序深度所占的百分比。</center>

<h2 id="小结"><a href="#小结" class="headerlink" title="小结"></a>小结</h2><p>测序质量的好坏直接影响下游的数据分析，对测序数据进行质控统计意义重大。GeneDock 公有云平台目前为 WES Germline，WGS Germline 和 WGS Somatic三套分析流程均提供了质控统计步骤，为您的数据分析提供前提保障。</p>

      
    </div>
    <footer class="article-footer">
      <a data-url="https://sun010848.github.io/2017/09/15/20170916_Sequencing_QC_Stat/" data-id="cj8wgtx390006c3s695i5cfs3" class="article-share-link">Share</a>
      
      
    </footer>
  </div>
  
</article>


  
    <article id="post-20170812_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER_week5" class="article article-type-post" itemscope itemprop="blogPost">
  <div class="article-meta">
    <a href="/2017/08/12/20170812_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER_week5/" class="article-date">
  <time datetime="2017-08-12T08:06:44.000Z" itemprop="datePublished">2017-08-12</time>
</a>
    
  </div>
  <div class="article-inner">
    
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2017/08/12/20170812_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER_week5/">Futurelean Clinical Bioinformatics-week5</a>
    </h1>
  

      </header>
    
    <div class="article-entry" itemprop="articleBody">
      
        <p>记录自己在学习《Clinical Bioinformatics》的点点滴滴思考。这次记录的是第五周的课程。<br>这周课程主要是讲临床场景方面的东西。</p>
<blockquote>
<p>This activity will see us return to St Mary’s hospital and the Manchester Centre for Genomic Medicine as we focus on the bioinformatician’s role in practice.</p>
</blockquote>
        
          <p class="article-more-link">
            <a href="/2017/08/12/20170812_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER_week5/#more">Read More</a>
          </p>
        
      
    </div>
    <footer class="article-footer">
      <a data-url="https://sun010848.github.io/2017/08/12/20170812_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER_week5/" data-id="cj8wgtx380005c3s62bltgiiw" class="article-share-link">Share</a>
      
      
    </footer>
  </div>
  
</article>


  
    <article id="post-20170812_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER" class="article article-type-post" itemscope itemprop="blogPost">
  <div class="article-meta">
    <a href="/2017/08/12/20170812_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER/" class="article-date">
  <time datetime="2017-08-12T07:15:34.000Z" itemprop="datePublished">2017-08-12</time>
</a>
    
  </div>
  <div class="article-inner">
    
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2017/08/12/20170812_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER/">Futurelean Clinical Bioinformatics-week4</a>
    </h1>
  

      </header>
    
    <div class="article-entry" itemprop="articleBody">
      
        <p>记录自己在学习《Clinical Bioinformatics》的点点滴滴思考。这次记录的是第四周的课程。<br>这周课程主要是讲医学伦理方面的东西。</p>
<blockquote>
<p>The focus of this week, therefore, will be on legalities, ethics and governance when dealing with genomic data</p>
</blockquote>
        
          <p class="article-more-link">
            <a href="/2017/08/12/20170812_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER/#more">Read More</a>
          </p>
        
      
    </div>
    <footer class="article-footer">
      <a data-url="https://sun010848.github.io/2017/08/12/20170812_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER/" data-id="cj8wgtx300003c3s61deu5bjy" class="article-share-link">Share</a>
      
      
    </footer>
  </div>
  
</article>


  
    <article id="post-20170731_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER" class="article article-type-post" itemscope itemprop="blogPost">
  <div class="article-meta">
    <a href="/2017/07/31/20170731_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER/" class="article-date">
  <time datetime="2017-07-31T13:13:42.000Z" itemprop="datePublished">2017-07-31</time>
</a>
    
  </div>
  <div class="article-inner">
    
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2017/07/31/20170731_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER/">Futurelean Clinical Bioinformatics-week3</a>
    </h1>
  

      </header>
    
    <div class="article-entry" itemprop="articleBody">
      
        <p>记录自己在学习《Clinical Bioinformatics》的点点滴滴思考。这次记录的是第三周的课程。</p>
        
          <p class="article-more-link">
            <a href="/2017/07/31/20170731_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER/#more">Read More</a>
          </p>
        
      
    </div>
    <footer class="article-footer">
      <a data-url="https://sun010848.github.io/2017/07/31/20170731_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER/" data-id="cj8wgtx2z0002c3s6rhybauei" class="article-share-link">Share</a>
      
      
    </footer>
  </div>
  
</article>


  
    <article id="post-20170725_WES_or_panel" class="article article-type-post" itemscope itemprop="blogPost">
  <div class="article-meta">
    <a href="/2017/07/25/20170725_WES_or_panel/" class="article-date">
  <time datetime="2017-07-25T14:37:11.000Z" itemprop="datePublished">2017-07-25</time>
</a>
    
  </div>
  <div class="article-inner">
    
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2017/07/25/20170725_WES_or_panel/">要panel还是Exome</a>
    </h1>
  

      </header>
    
    <div class="article-entry" itemprop="articleBody">
      
        <p>原文章标题《Targeted Panels or Exome – Which is the Right NGS Approach for Inherited Disease Research?》<br>地址：Targeted Panels or Exome – Which is the Right NGS Approach for Inherited Disease Research? - Behind the Bench</p>
        
          <p class="article-more-link">
            <a href="/2017/07/25/20170725_WES_or_panel/#more">Read More</a>
          </p>
        
      
    </div>
    <footer class="article-footer">
      <a data-url="https://sun010848.github.io/2017/07/25/20170725_WES_or_panel/" data-id="cj8wgtx2d0001c3s6h979bve2" class="article-share-link">Share</a>
      
      
    </footer>
  </div>
  
</article>


  
    <article id="postc-20170721_初探GATK4_Beta" class="article article-type-postc" itemscope itemprop="blogPost">
  <div class="article-meta">
    <a href="/2017/07/21/20170721_初探GATK4_Beta/" class="article-date">
  <time datetime="2017-07-21T03:00:00.000Z" itemprop="datePublished">2017-07-21</time>
</a>
    
  </div>
  <div class="article-inner">
    
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="article-title" href="/2017/07/21/20170721_初探GATK4_Beta/">初探 GATK4 beta</a>
    </h1>
  

      </header>
    
    <div class="article-entry" itemprop="articleBody">
      
        <p></p><p class="author-title">(本文是根据GeneDock基因数据工程师孙兴强对GATK4 beta版的调研结果整理，转载请保留转载信息和原文作者及链接。)</p><br><span style="font-size:14px"><p></p><p></p>
<h2 id="注：-本文是根据阅读GATK4在GitHub上的说明文档和版本开发记录并加上对该版本软件对生物信息研发和生产的一些体验组成的。GATK4-GitHub的官网内容会有引用标识，其余内容为笔者的理解内容。由于GATK4还在开发中，本文的时效性会受该项目的开发进度影响，请谨慎阅读。"><a href="#注：-本文是根据阅读GATK4在GitHub上的说明文档和版本开发记录并加上对该版本软件对生物信息研发和生产的一些体验组成的。GATK4-GitHub的官网内容会有引用标识，其余内容为笔者的理解内容。由于GATK4还在开发中，本文的时效性会受该项目的开发进度影响，请谨慎阅读。" class="headerlink" title="注： 本文是根据阅读GATK4在GitHub上的说明文档和版本开发记录并加上对该版本软件对生物信息研发和生产的一些体验组成的。GATK4 GitHub的官网内容会有引用标识，其余内容为笔者的理解内容。由于GATK4还在开发中，本文的时效性会受该项目的开发进度影响，请谨慎阅读。"></a><span style="font-size:16px">注： 本文是根据阅读GATK4在GitHub上的说明文档和版本开发记录并加上对该版本软件对生物信息研发和生产的一些体验组成的。GATK4 GitHub的官网内容会有引用标识，其余内容为笔者的理解内容。由于GATK4还在开发中，本文的时效性会受该项目的开发进度影响，请谨慎阅读。</span></h2>
        
          <p class="article-more-link">
            <a href="/2017/07/21/20170721_初探GATK4_Beta/#more">Read More</a>
          </p>
        
      
    </div>
    <footer class="article-footer">
      <a data-url="https://sun010848.github.io/2017/07/21/20170721_初探GATK4_Beta/" data-id="cj8wgtx290000c3s6zn4uljk8" class="article-share-link">Share</a>
      
      
    </footer>
  </div>
  
</article>


  

</section>
        
          <aside id="sidebar">
  
    

  
    

  
    
  
    
  <div class="widget-wrap">
    <h3 class="widget-title">Archives</h3>
    <div class="widget">
      <ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2017/10/">October 2017</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2017/09/">September 2017</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2017/08/">August 2017</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2017/07/">July 2017</a></li></ul>
    </div>
  </div>


  
    
  <div class="widget-wrap">
    <h3 class="widget-title">Recent Posts</h3>
    <div class="widget">
      <ul>
        
          <li>
            <a href="/2017/10/16/20171017_VEP_annotation2/">VEP--强大的变异注释工具</a>
          </li>
        
          <li>
            <a href="/2017/09/15/20170916_Sequencing_QC_Stat/">测序数据质控统计--微不足道？意义重大！</a>
          </li>
        
          <li>
            <a href="/2017/08/12/20170812_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER_week5/">Futurelean Clinical Bioinformatics-week5</a>
          </li>
        
          <li>
            <a href="/2017/08/12/20170812_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER/">Futurelean Clinical Bioinformatics-week4</a>
          </li>
        
          <li>
            <a href="/2017/07/31/20170731_learn_CLINICAL_BIOINFORMATICS_THE_UNIVERSITY_OF_MANCHESTER/">Futurelean Clinical Bioinformatics-week3</a>
          </li>
        
      </ul>
    </div>
  </div>

  
</aside>
        
      </div>
      <footer id="footer">
  
  <div class="outer">
    <div id="footer-info" class="inner">
      &copy; 2017 sun xingqiang<br>
      Powered by <a href="http://hexo.io/" target="_blank">Hexo</a>
    </div>
  </div>
</footer>
    </div>
    <nav id="mobile-nav">
  
    <a href="/" class="mobile-nav-link">Home</a>
  
    <a href="/archives" class="mobile-nav-link">Archives</a>
  
</nav>
    

<script src="//ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>


  <link rel="stylesheet" href="/fancybox/jquery.fancybox.css">
  <script src="/fancybox/jquery.fancybox.pack.js"></script>


<script src="/js/script.js"></script>

  </div>
</body>
</html>